jenkins-integration/beaker/install/shared/start_sut_background_scripts.rb

```diff
@@ -0,0 +1,28 @@
+require 'json'
+
+step "Launch background scripts on SUT" do
+  bg_scripts = ENV['SUT_BACKGROUND_SCRIPTS'].split("\n")
+  Beaker::Log.notify("Launching #{bg_scripts.count} background scripts")
+  master_tempdir = master.tmpdir('gplt_bg_scripts')
+  pids = {}
+  bg_scripts.each do |s|
+    script_filename = File.basename(s)
+    remote_path = File.join(master_tempdir, script_filename)
+
+    Beaker::Log.notify("Launching script '#{s}'")
+    scp_to(master, s, master_tempdir)
+    result = on(master, "#{remote_path} > #{master_tempdir}/stdout.log 2> #{master_tempdir}/stderr.log & echo $!")
+    Beaker::Log.notify("GOT RESULT FROM SCRIPT EXECUTION: '#{result}'")
```

Comment:
Is including the `result` object here useful?  When I ran this on my dev Jenkins instance, it just output an object with object id, like:

```
GOT RESULT FROM SCRIPT EXECUTION: '#<Beaker::Result:0x00000004fe81a8>'
```